app/helpers/form_helper.rb

```diff
@@ -204,6 +204,16 @@ def byte_size_f(f, attr, options = {})
     text_f(f, attr, options)
   end
 
+  def counter_f(f, attr, options = {})
+    options[:class] = options[:class].to_s + ' counter_spinner'
+    options[:help_block] ||= content_tag(:span, :class => 'maximum-limit hidden') do
+      content_tag(:i, '', :class => 'pficon-warning-triangle-o') +
+        content_tag(:span, ' ' + _('Specified value is higher than recommended maximum'), :class => 'error-message')
+    end
```

Comment:
these 4 lines are common with the byte_size_f above, consider extracting to a function